app/actions/__tests__/searchActions.spec.js

```diff
@@ -1,8 +1,8 @@
 import { expect } from 'chai';
 import simple from 'simple-mock';
 
-import searchActions from 'actions/searchActions';
-import apiUtils from 'utils/APIUtils';
+import { searchResources } from 'actions/searchActions';
+import * as apiUtils from 'utils/APIUtils';
```

Comment:
Oh, I guess the wildcard is needed because the utils doesn't have an export default. What's the reason for that?